src/os/ui/ogc/ogcserver.js

```diff
@@ -856,16 +857,24 @@ os.ui.ogc.OGCServer.prototype.parseWfsCapabilities = function(response, uri) {
     if (op) {
       var getFeatureEl = op.querySelector('Post');
       if (getFeatureEl != null) {
-        // Attr.value is the DOM4 property, while Attr.nodeValue inherited from Node should work on older browsers
-        var attr = getFeatureEl.attributes[0];
-        this.setWfsUrl(attr.value || attr.nodeValue);
+        if (getFeatureEl.hasAttributeNS(ol.format.XLink.NAMESPACE_URI, 'href')) {
+          this.setWfsUrl(getFeatureEl.getAttributeNS(ol.format.XLink.NAMESPACE_URI, 'href'));
+        } else {
+          var attr = getFeatureEl.attributes[0];
+          // Attr.value is the DOM4 property, while Attr.nodeValue inherited from Node should work on older browsers
+          this.setWfsUrl(attr.value || attr.nodeValue);
+        }
         this.setWfsPost(true);
       } else {
         getFeatureEl = op.querySelector('Get');
         if (getFeatureEl != null) {
-          // Attr.value is the DOM4 property, while Attr.nodeValue inherited from Node should work on older browsers
-          var attr = getFeatureEl.attributes[0];
-          this.setWfsUrl(attr.value || attr.nodeValue);
+          if (getFeatureEl.hasAttributeNS(ol.format.XLink.NAMESPACE_URI, 'href')) {
+            this.setWfsUrl(getFeatureEl.getAttributeNS(ol.format.XLink.NAMESPACE_URI, 'href'));
+          } else {
+            var attr = getFeatureEl.attributes[0];
+            // Attr.value is the DOM4 property, while Attr.nodeValue inherited from Node should work on older browsers
+            this.setWfsUrl(attr.value || attr.nodeValue);
+          }
```

Comment:
I was under the impression that we should replace the old method. Was there a case where that is still necessary?
```
this.setWfsUrl(getFeatureEl.getAttributeNS(ol.format.XLink.NAMESPACE_URI, 'href') || this.getWfsUrl());
```

Reply:
That would certainly simplify the logic, but there was some safety inherent in the old method in legacy cases where the capabilities document didn't strictly follow the schema. I'm happy to make the change if you're good elevating the strictness of the parsing like that!